ch01/playbooks/webservers.yml

```diff
@@ -5,7 +5,7 @@
     - name: Install nginx
       apt: name=nginx
     - name: install config file
-      template: src=nginx.conf.j2 dest=/etc/nginx/ngingx.onf
+      template: src=nginx.conf.j2 dest=/etc/nginx/ngingx.conf
```

Comment:
Thanks for the fix, but I actually mean that the file should be named `nginx.conf` instead of `ngingx.conf`, just like the src filename.

Not sure why I didn't notice the missing `c` lmao.

Reply:
Oh my!  Thanks for the check!